src/Masa.Utils.Exceptions/Results/DefaultExceptionResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc;
+
+public class DefaultExceptionResult : IActionResult
+{
+    public string? Message { get; set; }
+
+    public int StatusCode { get; set; }
+
+    public string ContentType { get; set; }
```

Comment:
Provide `ErrorCode`.

Reply:
But developer maybe don't use our I18n. Or use it for else business.

Reply:
Yes, but users will not use `DefaultExceptionResult`